Packs/VersaDirector/Integrations/VersaDirector/VersaDirector_description.md

```diff
@@ -0,0 +1,39 @@
+# Versa Director
+
+## Versa Director Authentication flow:
+
+- **Versa Director Integration can work in 2 authentication methods: Basic or Auth Token.**
+- **By default, Auth Token Authentication will be used in the integration, to use basic authentication see the Basic Authentication section**
+
+### Basic Authentication:
+
+Basic authentication method based on ***Username*** and ***Password*** parameters (default authentication method).
```

Comment:
Is it the default method if the use_basic_auth param is false by default?

Reply:
This has been fixed locally and will be updated in the next pr